src/main.h

```diff
@@ -10,7 +10,7 @@
 
 #include "AdaptiveByteStream.h"
 #include "SSD_dll.h"
-#include "common/AdaptiveStream.h"
+
 #include "common/AdaptiveTree.h"
 #include "common/Chooser.h"
 #include "samplereader/SampleReader.h"
```

Comment:
also here we need to keep only the required includes for the definitions of classes